src/backend/prisma/seed.ts

```diff
@@ -17,6 +18,10 @@ const performSeed: () => Promise<void> = async () => {
     await prisma.user.create({ data: { ...seedUser } });
   }
 
+  for (const seedRisk of dbSeedAllRisks) {
+    await prisma.risk.create({ data: { ...seedRisk } });
```

Comment:
Pretty sure you're gonna need to do a connect to link it up with the user creating it. See change requests for an example probably

Reply:
Also might make more sense to have this come after projects

Reply:
And probably connect to the project